OpenData/secular printed material - 1921 - 1990/Coraa ny Gael/Coraa ny Gael 1953-08-25/document.csv

```diff
@@ -1,31 +1,31 @@
+T’eh er-chee shassoo son ny Fir-lhee Americanagh ec chaglym vooar dy Fir-lhee ayns Lunnin.,He is about to represent the American Doctors at a big meeting of doctors in London.,
 "Keayrt dy row va eh-hene yn Reiltagh jeh’n Cheshaght dy Vanninee ayns America Twoaieagh. Laa ennagh er y chiaghtyn shoh chaie, hug eh shilley er Purt ny h-inshey, y voayl haink eh voish hoshiaght, as v’eh er ny oltaghey liorish yn ardhoieder ny varrantee ny valley. Mnr. Jamys H. Kelly.","At a certain time he himself was President of the North American Manx Association. Some day last week he went to see Peel, the place he first came from, and he was welcomed by the chairman of the town commissioners, Mr. James H. Kelly.",
-"     Tra hug Fer Lhee Moore booise daue er son y gein-vie dy dooar eh aynshen, dooyrt eh “Ta’n jishag aym cliaghtit dy rouail ooilley mygeayrt America, as yinnagh eh gra nagh row boayll erbee feeu son veg, mannagh row Manninagh beaghey aynshen, son va dy-chooilley lught-thie Vanninagh jannoo feer vooar jeh dagh Vanninagh elley yinnagh goll dy chur shilley orroo.’","     When Doctor Moore gave thanks to them for the good cheer he received there, he said “My dad is used to wandering all around America, and he would say there wasn’t any spot worth anything, unless there was a Manx person living there, because every Manx family were making a great deal of every other Manx person who would go to visit them.’",
-     Ta shin clashtyn dy vel Harry Johnson yn canteyder cheet-er dy mie lurg yn chingys vooar v’echey dy gerrid. V’eh giarit lesh skynn y Fer-lhee ayns y thie-lheeys.,     We hear that Harry Johnson the auctionerr is coming on well after the great illness he had recently. He was cut by the knife of the doctor (operated on) in the hospital.,
+"     Tra hug Fer Lhee Moore booise daue er son y gein-vie dy dooar eh aynshen, dooyrt eh “Ta’n jishag aym cliaghtit dy rouail ooilley mygeayrt America, as yinnagh eh gra nagh row boayll erbee feeu son veg, mannagh row Manninagh beaghey aynshen, son va dy-chooilley lught-thie Vanninagh jannoo feer vooar jeh dagh Vanninagh elley yinnagh goll dy chur shilley orroo.’","     When Doctor Moore gave thanks to them for the good cheer he received there, he said ‘My dad is used to wandering all around America, and he would say there wasn’t any spot worth anything, unless there was a Manx person living there, because every Manx family were making a great deal of every other Manx person who would go to visit them.’",
```

Comment:
```suggestion
"     Tra hug Fer Lhee Moore booise daue er son y gein-vie dy dooar eh aynshen, dooyrt eh “Ta’n jishag aym cliaghtit dy rouail ooilley mygeayrt America, as yinnagh eh gra nagh row boayll erbee feeu son veg, mannagh row Manninagh beaghey aynshen, son va dy-chooilley lught-thie Vanninagh jannoo feer vooar jeh dagh Vanninagh elley yinnagh goll dy chur shilley orroo."","     When Doctor Moore gave thanks to them for the good cheer he received there, he said "My dad is used to wandering all around America, and he would say there wasn’t any spot worth anything, unless there was a Manx person living there, because every Manx family were making a great deal of every other Manx person who would go to visit them."",
```

Reply:
I don't think the double quotes work, single inside double is fine for quoted speech. Or have I misunderstood what's going on here?

Reply:
Hi Jane,
Thanks so much again for your professional-level help, it's really appreciated. I hope I am not being too heavy-handed or OCD here but the thing is: the outside level of quotes is not a quotation at all, it is simply something to tell the parser of a comma delimited or CSV file (Comma Separated Values) that text with a comma in the middle of it is still a single string. This is what the final version looks like on the screen:
<img width="1137" alt="Screenshot 2024-05-05 at 9 46 03 AM" src="https://github.com/david-allison/manx-search-data/assets/47546076/d7f669e6-27ee-4497-934b-dce8183437c2">

Ken

Reply:
Here is the original Manx in its original appearance:
<img width="459" alt="Screenshot 2024-05-05 at 4 41 07 PM" src="https://github.com/david-allison/manx-search-data/assets/47546076/0212d0d0-98e5-4da7-91c6-75ea88413cc4">

Reply:
Urgh, yes, of course it is. I had understood how the quotation marks work, but for some reason I thought the Doctor Moore quote was inside a quotation. Sigh. It'll sink in soon! Thank you for your patience.